### DISCUSSION TITLE
create a view from two different tables from two different resource groups - Azure Postgres

### DISCUSSION THREAD

**Discussion by jgrover21** [dbeaver/dbeaver#20995] [Q&A] [answered]
### Description

I am unable to create a view from two different tables from two different resource groups and servers.  can I do that?

### DBeaver Version

23.1.1

### Operating System

windows

### Database and driver

postgres 14.0

**Comment by E1izabeth** [comment, marked as answer]
As far as I know, DBeaver doesn't provide such possibility.
However, you may check if it's possible using dblink extension and writing the corresponding queries.
